src/notification.cpp

```diff
@@ -339,12 +340,12 @@ void Notification::mouseReleaseEvent(QMouseEvent * event)
             QWidget::mouseReleaseEvent(event);
             return;
         }
-        if (appName == appLabel->text() || windowTitle == appLabel->text())
-        {
-            KWindowSystem::raiseWindow(i);
-            closeButton_clicked();
-            return;
-        }
+ //       if (appName == appLabel->text() || windowTitle == appLabel->text())
```

Comment:
This I wasn't able to fix, `activateWindow` wasn't accepted.

Reply:
Try to change it to

```c++
        if (appName == appLabel->text() || windowTitle == appLabel->text())
        {
            if (auto w = QWidget::find(i))
                w->raise(i);
            closeButton_clicked();
            return;
        }
```